sql-encryption-protector-byok/metadata.json

```diff
@@ -1,9 +1,10 @@
 {
-    "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
-    "type": "QuickStart",
-    "itemDisplayName": "Create an Azure SQL Server, with data encryption protector",
-    "description": "This template creates an Azure SQL server, activates the data encryption protector using a given key stored in a given Key Vault",
-    "summary": "This template creates an Azure SQL server with data encryption protector",
-    "githubUsername": "jchomarat",
-    "dateUpdated": "2019-07-08"
-  }
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "type": "QuickStart",
+  "itemDisplayName": "Create an Azure SQL Server, with data encryption protector",
+  "description": "This template creates an Azure SQL server, activates the data encryption protector using a given key stored in a given Key Vault",
+  "summary": "This template creates an Azure SQL server with data encryption protector",
+  "githubUsername": "jchomarat",
+  "dateUpdated": "2020-09-24",
+  "validationType": "Manual"
```

Comment:
This doesn't need manual validation in needs a prereq, similar to this one: https://github.com/Azure/azure-quickstart-templates/blob/master/201-machine-learning-encrypted-workspace/prereqs/prereq.azuredeploy.json

Would be great if you could add it, but in either case remove the manual validation tag since this actually can be automated.

Reply:
Done!